### PR TITLE
Two issues: use "node ci" instead of "node install", and issue warning about bad paths installed by pipenv.

### PR DESCRIPTION
**Pull request by john-sifive**
Pipenv  uses "realpafh" to generate internal file paths - unfortunately, "realpath" generates invalid paths when run under Wake/Fuse. This fix restores the "setLocalOnly" mode when running pipenv under wake.

Additionally, use the "npm ci" command to install node modules.  Unlike the normal "install" command, "ci" does not update the lock file.  Updates to the lock file generate Wake errors.

### CODE REVIEW

File: python.wake [hunk 1]
```diff
@@ -82,13 +97,14 @@ target installPipenvEnv pipDir =
...
 
+  # Create a plan to run the script and create a virtual environment.
   makePlan args visible
   | setPlanResources ("python/python/3.7.1", Nil)
-  | runJob
-  | getJobOutputs
-
 
+  # Disable Fuse since it is incompatible with pipenv.
+  | setPlanLocalOnly True
```
Comment (richardxia) [inline python.wake]: I'm worried about this line because setting a plan to be local-only has a very powerful and contagious effect, preventing other commands from running correctly. I feel the real problem is with how the Wake preload runner works, since this is a "leaky" part of the abstraction.

Can we back out of this half of the change and restrict this PR to only updating the `npm` command? We can file a GitHub issue on this repository to track and discuss the other issue.

File: python.wake [hunk 2]
```diff
@@ -38,6 +38,7 @@ global def addPythonEnv pipDir plan =
   | editPlanVisible (installed ++ _)
   | addPlanRelativePath  "PYTHONPATH"  venv
   | addPlanRelativePath  "PATH" "{venv}/bin"
+  | setPlanEnvironmentVar "VIRTUAL_ENV"  venv
```
Comment (richardxia) [inline python.wake]: Can you revert this change as well so that we only tackle all of it at once as opposed to having some of the changes within this PR?

Comment (john-sifive) [reply python.wake]: Shall do.

**Comment by john-sifive** [comment]
As an additional comment, the issues we're encountering with "realpath" and pipenv are going to be  a problem for deputy as well, whether we setLocalOnly or not.  Wes had originally made the comment we should rewrite pipenv to make it conform, which I admittedly did not take seriously at the time. I think I'm beginning to agree.  While we can't easily rewrite pipenv (or conda), we could do additional processing to the "bin" wrappers it produces, or in the extreme case put wrappers around the bin wrappers.  This is just brainstorming, but if we want to support Deputy with remote execution, then having "movable" workspaces is essential.  This is essentially the  condition Wes was stating.  (Did I get it right?)
